pysparkling/tests/test_resolve_filenames.py

```diff
@@ -68,6 +68,8 @@ def test_s3_1():
 
 
 def test_hdfs_resolve_filenames_with_wildcard():
+    # hdfs is an optional dependency
+    # pylint: disable=C0415
```

Comment:
I'd prefer the human readable version here.

Reply:
Oh yes sure, I didn't know about it when I added the line but it's much better indeed, done!